pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -0,0 +1,71 @@
+{ alsa-lib
+, cups
+, dpkg
+, fetchurl
+, gjs
+, glib
+, gtk3
+, gtk4
+, lib
+, libdrm
+, libgcrypt
+, libkrb5
+, mesa # for libgbm
+, nss
+, xorg
+, systemd
+, stdenv
+, at-spi2-core
+, autoPatchelfHook
+, wrapGAppsHook
+}:
+
+let
+  mirror = "https://dldir1.qq.com/qqfile/qq/QQNT/50eed662";
```

Comment:
This is used only once. Probably better to include it in `url` and make the structure simpler.

Reply:
will be used more if we add arm64 support